src/routes/RepresentativeRoute/RepresentativeRoute.js

```diff
@@ -41,8 +46,10 @@ export default class RepresentativeRoute extends React.Component {
           <h1>{name}</h1>
           <h2>{currentRole}</h2>
           <h3>{party}</h3>
-          <p>Phone: {phone}</p>
-          <p>Website: <a href={url}>{url}</a></p>
+          {phone && <p>Phone: {phone}</p>}
+          {url && <p>Website: <a href={url}>{url}</a></p>}
+          {twitterUrl && <a href={twitterUrl}><Icon name='twitter'/></a>}
+          {fbUrl && <a href={fbUrl}><Icon name='facebook'/></a>}
```

Comment:
what does the && do?

Reply:
> what does the && do?

If the field exists, i.e. not null, display what is after the &&. If its null, stop the statement and move on. False && ...     would stop before the element

Reply:
Clever